hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/AbstractScheduleHandler.java

```diff
@@ -78,6 +87,14 @@ protected Set<Key> allKeysForTransaction(
         return getKeySetFromTransactionKeys(keyStructure);
     }
 
+    /**
+     * Get the schedule keys result to sign the transaction
+     *
+     * @param scheduleInState the schedule in state
+     * @param context         the Prehandle context
+     * @return the schedule keys result containing the updated signatories and the remaining required keys
+     * @throws HandleException the handle exception
```

Comment:
Same comment as `@throws PreCheckException` above. There seem to be a few of these.